app/views/two_factor_mailer/backup_code.html.erb

```diff
@@ -0,0 +1,21 @@
+<p>Hi <%= @user&.username %>,</p>
+
+<h3>Your two-factor authentication backup code</h3>
+
+<p>
+  You're receiving this email because you have two-factor authentication enabled on your Codidact network account. We've
+  recently added backup codes for every account using 2FA, so that you can still log in if you lose access to your
+  authenticator app.
+</p>
+
+<p>
+  Your backup code can be found in your user account settings, under
+  <a href="<%= two_factor_status_path %>">Two-Factor Authentication settings</a>. Please save it somewhere safe.

```

Comment:
After testing this a bit more thoroughly it turns out that one has to use url and not path to get a valid link in the mailers. Apologies for the mistake.

```suggestion
  <a href="<%= two_factor_status_url %>">Two-Factor Authentication settings</a>. Please save it somewhere safe.

```